app/routes.js

```diff
@@ -15,6 +15,8 @@ const addContext = ({ context, user, csrfToken }) => ({
 export const routes = (authProvider) => {
   const router = express.Router();
 
+  healthRoutes({ router, dependencies: getHealthCheckDependencies(config), logger });
```

Comment:
what are you getting from the config? how does the healthcheck routes know how unhealthy or degraded dependencies affects it? for this app it will be unhealthy if either isapi or ordapi are down but what about pb when it will be unhealthy if bapi is down but not if dapi or isapi are down?

Reply:
There is flag that is passed in as part of dependencies that will tell the healthRoutes if that dependency is critical or not. If one of these critical dependencies fails it is `unhealthy`. If it isn't then the ready status is `degraded`

Reply:
ah yeah I was misunderstanding soz